include/sys/zio.h

```diff
@@ -167,6 +167,12 @@ enum zio_encrypt {
 #define	ZIO_FAILURE_MODE_CONTINUE	1
 #define	ZIO_FAILURE_MODE_PANIC		2
 
+enum zio_suspend_reason {
+	ZIO_SUSPEND_NONE = 0,
+	ZIO_SUSPEND_IOERR,
+	ZIO_SUSPEND_MMP,
+};
```

Comment:
Please add `zio_suspend_reason_t` as a typedef and use that throughout.

Reply:
Fixed